exdir/core/group.py

```diff
@@ -509,20 +542,22 @@ def __len__(self):
         assert_file_open(self.file)
         return len([a for a in self])
 
-    def get(self, key):
+    def get(self, name, get_link=False):
```

Comment:
I think the getter should first try to see if it is an object or a link, instead of forcing the user to tell it to expect a link or not. This is also how h5py works.